src/index.ts

```diff
@@ -49,6 +51,32 @@ async function main() {
     logger.error({ exception }, 'API was unable to start')
   }
 
+  const batchWriter = new BatchWriter({
```

Comment:
@lautarodragan should we wrap batchWriter initialization with an if statement that checks for `enableTimestamping` like the blockchainWriter? I don't think there is any reason to publish batches if `enableTimestamping` is false.

Reply:
Sounds good to me